nf_core/components/create.py

```diff
@@ -394,18 +391,12 @@ def _get_component_dirs(self):
 
             # Set file paths
             # For modules - can be tool/ or tool/subtool/ so can't do in template directory structure
-            file_paths[os.path.join(self.component_type, "main.nf")] = os.path.join(component_dir, "main.nf")
-            file_paths[os.path.join(self.component_type, "meta.yml")] = os.path.join(component_dir, "meta.yml")
+            file_paths["main.nf"] = component_dir / "main.nf"
+            file_paths["meta.yml"] = component_dir / "meta.yml"
```

Comment:
```suggestion
            file_paths["main.nf"] = Path(component_dir,"main.nf")
            file_paths["meta.yml"] = Path(component_dir,"meta.yml")
```

Reply:
I don't think we need to explicitly add `Path()` everywhere, or is there a reason that I am missing?

Reply:
I think I am now just used to overuse Path() 🙂  if you prefer the other way (and it is a bit more readable), it's also totally fine for me (and with type hints we should get an error message, when we try to concat a string with a string)

Reply:
I will keep it with `/` then if that's ok 😄